src/TorchSharp/TorchSharp.csproj

```diff
@@ -23,6 +23,8 @@
   <ItemGroup>
     <PackageReference Include="Google.Protobuf" Version="3.21.5" />
     <PackageReference Include="SharpZipLib" Version="1.3.3" />
+    <PackageReference Include="SkiaSharp" Version="2.88.3" />
+    <PackageReference Include="SkiaSharp.NativeAssets.Linux.NoDependencies" Version="2.88.3" />
```

Comment:
>SkiaSharp.NativeAssets.Linux.NoDependencies [](http://example.com/codeflow?start=31&length=43)

@NiklasGustafsson what about Windows and MacOS dependency? or we don't run there?

Reply:
Those are actually included in the SkiaSharp package. Only the Linux natives are outside of it.